_posts/2023-03-11-Adding-a-Timeout-feature-to-Rustgem.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Adding a Timeout feature to Rustgem"
+date: 2023-03-11
+author: Maíra Canal
+layout: post
+permalink: /adding-timeout-rustgem/
+categories: Genel
+tags: [igalia, graphics]
+---
+
+After my last blogpost, I kept developing the Rust for VGEM driver, also known
+as rustgem, for now. Previously, I have developed two important features of the
```

Comment:
Unclear if the "for now" referes to the name being temporary, in which case I'd remove the comma, or if you're refering to your current general plans. In that case, I'd remove "for now".

```suggestion
as rustgem for now. Previously, I had developed two important features of the
```